presto-spi/src/main/java/com/facebook/presto/spi/type/AbstractVariableWidthType.java

```diff
@@ -21,7 +21,7 @@
         extends AbstractType
         implements VariableWidthType
 {
-    private static final int EXPECTED_BYTES_PER_ENTRY = 32;
+    public static final int EXPECTED_BYTES_PER_ENTRY = 32;
```

Comment:
`com.facebook.presto.spi.block.VariableWidthBlockBuilder` also accesses this, so leaving this as `public`.